commerce-product-definitions-web/src/main/java/com/liferay/commerce/product/definitions/web/internal/portlet/AssetCategoryFriendlyURLResolver.java

```diff
@@ -140,7 +140,7 @@ public String getActualURL(
 			assetCategory.getDescription(languageId), httpServletRequest);
 
 		List<AssetTag> assetTags = _assetTagLocalService.getTags(
-			AssetCategory.class.getName(), assetCategory.getPrimaryKey());
+			AssetCategory.class.getName(), assetCategory.getClassPK());
```

Comment:
@ethanbustad This should have been classPK aswell according to [javadoc](https://docs.liferay.com/portal/7.0/javadocs/portal-kernel/com/liferay/asset/kernel/service/AssetTagLocalServiceUtil.html#getTags-java.lang.String-long-)

Reply:
disregard